backend/app/models/ability.rb

```diff
@@ -0,0 +1,35 @@
+class Ability
+  include CanCan::Ability
+
+  attr_accessor :user
+
+  def initialize(user)
+    @user = user
+
+    default_rights
+    user_rights if user.present?
+  end
+
+  private
+
+  def default_rights
+    # only data from approved users are visible
+    can %i[index show], ProjectDeveloper, account: {review_status: :approved}
+    can %i[index show], Investor, account: {review_status: :approved}
+    can %i[index show], Project, project_developer: {account: {review_status: :approved}}
+    can %i[index show], OpenCall, investor: {account: {review_status: :approved}}
+  end
+
+  def user_rights
```

Comment:
I think we need to also make sure that only approved `Investor` and `ProjectDeveloper` can create/update projects and open calls.

Reply:
I think UI prevents creating/updating projects, waiting for approval, please correct me if not. We can make it consistent in this way.

Reply:
I have checked code and don't see such limitation -- even unapproved Investor/Project Developer can create OpenCall/Project. I have also ticket https://vizzuality.atlassian.net/browse/LET-513 which specifies that Projects/OpenCalls from unapproved Investors/ProjectDevelopers are hidden by default, but there is no word that unapproved users cannot create/update it at all

Reply:
Maybe it was forgotten, what do you think @agnessa? Can unapproved accounts do anything on the platform?

Reply:
do we? that's not in acceptance criteria :) I'm not sure about that actually

Reply:
Ok. Let's then ask questions and add a new story if needed.